_themes/launcher.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
```

Comment:
Okay.  I won't be able to until this afternoon (morning here), so check back later.

I'll definitely make a point to get this added before tomorrow though. :+1: